python/ray/_private/runtime_env/working_dir.py

```diff
@@ -371,18 +398,20 @@ def upload_runtime_env_package_if_needed(self, job_config: JobConfig):
                 working_dir = job_config.runtime_env.get("working_dir")
                 py_modules = job_config.runtime_env.get("py_modules")
                 excludes = job_config.runtime_env.get("excludes") or []
-                _logger.info(
-                    f"{pkg_uri} doesn't exist. Create new package with"
-                    f" {working_dir} and {py_modules}")
+                logger.info(f"{pkg_uri} doesn't exist. Create new package with"
+                            f" {working_dir} and {py_modules}")
                 if not pkg_file.exists():
                     create_project_package(working_dir, py_modules, excludes,
                                            file_path)
```

Comment:
Can we also pipe the logger through the `create_project_package` and `_zip_module` call path in this PR?  It might be easy to forget later otherwise

Reply:
done